independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/tooling/dockerfiles/base/src/main/docker/Dockerfile.tpl.qute.multistage.native-micro

```diff
@@ -0,0 +1,39 @@
+####
```

Comment:
Adding yet another docker file?

Reply:
> Sure but how will they know how to use those files ?

By specifying the Dockerfile path (eg. https://docs.digitalocean.com/products/app-platform/reference/dockerfile/#defining-the-dockerfile)

Reply:
> Adding yet another docker file?

I followed the pattern we already have with the native Dockerfiles but I am fine with just one native Multi staged Dockerfile